libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFSDKWebViewStateManager.h

```diff
@@ -24,7 +24,18 @@
 #import <Foundation/Foundation.h>
 
 @class WKProcessPool;
-@interface SFSDKWKProcessPoolFactory : NSObject
+
+@interface SFSDKWebViewStateManager : NSObject
+
+/**
+  Reset some associated state with WKWebView and UIWebView e.g. Remove Cookies then add SID cookie
+ */
++ (void)resetSessionWithAccessToken:(NSString *) accessToken andProtocol:(BOOL) isSecure;
```

Comment:
@khawkins I meant to change that after first iteration on my end. Guess it slipped. I will make that change.